lib/house.rb

```diff
@@ -0,0 +1,62 @@
+class House
+attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    rooms << room
+  end
+
+  def above_market_average?
+    if @price[1..-1].to_i > 500000
```

Comment:
Nice job with the range to get rid of the `$`. This should have been done in the initialize method, so that price is stored as an integer every time an object is instantiated. This way it's formatted for calculations from that point onward. 

This method could also be simplified to `@price[1..-1].to_i > 500000` which would return the correct boolean.